package.json

```diff
@@ -49,28 +49,64 @@
   },
   "scripts": {
     "bootstrap:hoist": "lerna bootstrap --hoist --force-local",
-    "clean": "lerna clean --yes && rm -rf node_modules || true",
     "clean:install": "rm package-lock.json; npm run clean && npm install",
+    "clean": "lerna clean --yes && rm -rf node_modules || true",
+    "compile:prod": "rm -rf ./build; NODE_ENV=production webpack -p --config webpack.dev-site.config.js",
     "compile": "lerna run compile --parallel",
     "danger": "danger ci",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:prod",
+    "heroku-prebuild": "npx lerna init",
     "jest": "jest",
-    "mocha": "cd packages/terra-functional-testing && npm run mocha",
-    "lint": "npm run lint:js",
     "lint:js": "eslint --ext .js,.jsx .",
+    "lint": "npm run lint:js",
+    "mocha": "cd packages/terra-functional-testing && npm run mocha",
     "pack": "webpack -p",
     "postinstall": "link-parent-bin && npm run bootstrap:hoist",
+    "start": "webpack-dev-server --config webpack.dev-site.config.js",
+    "start-heroku": "node ./scripts/hack-express-server.js --port $PORT --site './build' ",
     "test": "npm run lint && npm run jest && npm run mocha && npm run wdio",
-    "wdio": "rm -rf build && npm run pack && npm run wdio-site && npm run wdio-webpack",
+    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js",
     "wdio-site": "SITE=build node ./packages/terra-functional-testing/bin/test-runner.js",
     "wdio-webpack": "node ./packages/terra-functional-testing/bin/test-runner.js",
-    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
+    "wdio": "rm -rf build && npm run pack && npm run wdio-site && npm run wdio-webpack"
   },
   "devDependencies": {
+    "@babel/cli": "^7.10.5",
+    "@babel/core": "^7.11.1",
+    "@babel/helpers": "^7.10.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
+    "@babel/plugin-syntax-async-generators": "^7.8.4",
+    "@babel/plugin-transform-regenerator": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.11.0",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.10.5",
+    "@wdio/cli": "^6.3.6",
+    "@wdio/local-runner": "^6.3.6",
+    "@wdio/mocha-framework": "^6.3.6",
+    "@wdio/selenium-standalone-service": "^6.1.14",
+    "@wdio/sync": "^6.3.6",
+    "browserslist-config-terra": "^1.2.0",
+    "chai": "^4.2.0",
+    "commander": "^5.1.0",
+    "core-js": "^3.1.3",
     "danger": "^9.1.0",
     "eslint": "^6.1.0",
+    "glob": "^7.1.6",
     "html-webpack-plugin": "^4.3.0",
     "jest": "^26.0.1",
     "lerna": "^3.20.2",
-    "link-parent-bin": "^1.0.0"
+    "link-parent-bin": "^1.0.0",
+    "mocha": "^8.1.1",
+    "nock": "^12.0.3",
+    "np": "^6.3.2",
+    "react-dom": "^16.8.5",
+    "react": "^16.8.5",
+    "regenerator-runtime": "^0.13.2",
+    "sinon": "^9.0.2",
+    "terra-dev-site": "^6.5.0",
```

Comment:
This causes an unresloved peer on terra-toolkit... it's ok though because we use the new webpack config instead of the terra-toolkit one.